ada-project-docs/wave_03.md

```diff
@@ -34,6 +34,12 @@ when I send a `PATCH` request to `/tasks/1/mark_complete`,
 
 then the task is updated, so that its `completed_at` value is the current date, and I get this response:
 
+`204 No Content`
+
+The response should have a mimetype of "application/json" to keep our API response type consistent.
+
+If after the `PATCH` request I fetch the task by sending a `GET` request to `/tasks/1`, I get this response:
```

Comment:
We keep the preceding slash and use code formatting in the other route examples through the waves, so I want to leave that in on `/tasks/1`, and it looks like we lost the ending code tic around the route, so just making a gentle modification to your suggestion.

```suggestion
After I have made the 'PATCH' request, I can submit a 'GET' request to `/tasks/1`, which will return the response:
```